src/custom.d.ts

```diff
@@ -0,0 +1,14 @@
+import { UserDocument } from '@/model/auth/user';
+
+declare global {
+  namespace Express {
+    interface Request {
+      /**
+       * The authenticated user of the request.
```

Comment:
```suggestion
       * The request from authenticaed user.
```

Reply:
No, it doesn't refer to the request. What the comment talks about is the user (account) that requested.

Reply:
done